go/migration/migration.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Package migration provides functions to convert Noms values between different data versions.
```

Comment:
s/data versions/Noms format versions/

(and maybe link to information about the format version)